src/test/java/com/wikia/webdriver/testcases/mercurytests/InteractiveMapsTests.java

```diff
@@ -82,8 +81,8 @@ public void MercuryInteractiveMapsTest_001_MapModal_Url_Title_PinPopUp_Close() {
     );
   }
 
-  // IMAPT02
-  @Test(groups = {"MercuryInteractiveMapsTest_002", "MercuryInteractiveMapsTests", "Mercury"})
+  @Test(groups = "MercuryInteractiveMapsTest_002")
+  @Execute(browser = Browser.ANDROID)
```

Comment:
android is not a browser it's an OS. My proposal is `driver = Drivers.ANDROID_APPIUM`